eclass/unpacker.eclass

```diff
@@ -429,7 +429,21 @@ _unpacker_get_decompressor() {
 	*.lzma|*.xz|*.txz)
 		echo "xz -T$(makeopts_jobs) -dc" ;;
 	*.lz)
-		: ${UNPACKER_LZIP:=$(type -P plzip || type -P pdlzip || type -P lzip)}
+		find_lz_unpacker() {
+			local has_version_arg="-b"
+
+			[[ ${EAPI} == 6 ]] && has_version_arg="--host-root"
+			if has_version "${has_version_arg}" ">=app-arch/xz-utils-5.4.0" ; then
+				echo xz && return
```

Comment:
```suggestion
				echo xz
				return
```

I suppose we don't want to give the wrong suggestion that we account for `echo` failing ;-).